L1Trigger/L1TMuonBarrel/src/L1TMuonBarrelParamsHelper.cc

```diff
@@ -35,8 +35,8 @@ void L1TMuonBarrelParamsHelper::configFromPy(std::map<std::string, int>& allInts
 
 	setAssLUTPath(AssLUTpath);
 	///Read Pt assignment Luts
-	std::vector<LUT> pta_lut(0); pta_lut.reserve(19);
-	std::vector<int> pta_threshold(6); pta_threshold.reserve(10);
+	std::vector<LUT> pta_lut(19);
```

Comment:
@rekovic - careful - I believe that pta_lut should be reserve not resize - see that there is a push_back (but only 1 push_back.. )  on line 379 (this code is extremely confusing)

please confirm...